x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/scalar/multivalue/AbstractMultivalueFunctionTestCase.java

```diff
@@ -314,7 +325,64 @@ protected static void longs(
     }
 
     /**
-     * Build a test case with unsigned {@code long} values.
+     * Build many test cases with {@code long} values.
```

Comment:
I think there may be some copy-paste mistakes here.
```suggestion
     * Build many test cases with @{code date} values.
```
Below, the test case names also generally use `long` instead of `datetime`, which clashes with the test names used above in `longs`.

Or is that in purpose because we represent longs as datetimes? If so, the javadoc + test case names may be a bit confusing.

Reply:
Nah, I should just call it `date` or `datetime`.